mobile/library/common/config/config.cc

```diff
@@ -538,5 +539,27 @@ stats_sinks: *stats_sinks
 R"(
         overload:
           global_downstream_max_connections: 0xffffffff # uint32 max
+#{custom_layers}
 )";
+
+const char* rtds_layer_insert = R"(
+    - name: {}
+      rtds_layer:
+        name: {}
+        rtds_config:
+          initial_fetch_timeout:
+            seconds: {}
+          resource_api_version: V3
+          ads: {{}})";
+
+const char* ads_insert = R"(
```

Comment:
same, i don't think we need this anymore, right?

Reply:
Same reply, both builders are currently in use. We are using the yaml approach but verifying it matches against the protobuf.